src/viam/robot/client.py

```diff
@@ -284,10 +286,16 @@ def get_component(self, name: ResourceName) -> ComponentBase:
         Returns:
             ComponentBase: The component
         """
-        if name.type != "component":
-            raise ViamError(f"ResourceName does not describe a component: {name}")
+        if name.type != RESOURCE_TYPE_COMPONENT:
+            raise ValueError(f"ResourceName does not describe a component: {name}")
         with self._lock:
-            return self._manager.get_component(ComponentBase, name)
+            return self._manager.get_resource(ComponentBase, name)
+
+    def get_service(self, name: ResourceName) -> ServiceBase:
```

Comment:
Should this have similar documentation to the `get_component` func?

Reply:
You are absolutely right. I'll add it now